spec/api/v1/notifications_controller_spec.rb

```diff
@@ -1,5 +1,83 @@
 require 'rails_helper'
 
 RSpec.describe "Notifications API", type: :api do
-  it "needs some tests"
+  let!(:user) { create(:user_with_team) }
+  let(:team) { user.teams.first }
+
+  let!(:notification_1) { create(:notification, user: user, team: team, action_performed: "created", seen: false) }
```

Comment:
there’s a `create_list` factory girl helper that creates a given amount of records for you